metrics-operator/pkg/metrics/server.go

```diff
@@ -187,7 +187,10 @@ func (m *serverManager) recordMetrics() {
 						Name: normName,
 						Help: metric.Name,
 					})
-					prometheus.MustRegister(m.metrics.gauges[normName])
+					err := prometheus.Register(m.metrics.gauges[normName])
```

Comment:
@bacherfl I think we should not panic the operator if two metrics are registered with the same name